README.md

```diff
@@ -41,14 +41,31 @@ To install or update dependencies before making front end changes:
 $ yarn install
 ```
 
-The Tailwind CSS library is used to add CSS styles to the site. Tailwind automatically builds an output.css file which is the main stylesheet for the website. 
+The Tailwind CSS library is used to add CSS styles to the site. Tailwind automatically builds an output.css file which is the main stylesheet for the website.
 
-In a separate terminal window the following command will watch for any CSS changes and automatically rebuild the output.css file: 
+In a separate terminal window the following command will watch for any CSS changes and automatically rebuild the output.css file:
 
 ```
 $ yarn dev:css
 ```
 
+## Data Playground
+
+If you add a source to the Data Playground, remember to run the indexing command to ensure it shows up in the search functionality:
+
+```
+$ yarn index
+```
+
+**REMEMBER** after running this command, please go to `static/index.json` and delete lines 2-6 so they don't get added/committed:
```

Comment:
Should lines `254 - 278` of this file be removed or should we have the info listed more than once?